### DISCUSSION TITLE
Cannot publish npm and submit git

### DISCUSSION THREAD

**Discussion by 1977474741** [semantic-release/semantic-release#2599] [Commit Convention] [answered]
My submission information is as follows: "bugfix: fix a bug". Why doesn't he update the version? Only changing the type to feat can he go through the version update process. Why

**Comment by travi** [comment, marked as answer]
`bugfix` is not a valid commit type for the default commit convention. you are likely wanting `fix: fix a bug` instead

**Reply by 1977474741** [reply]
> `bugfix`不是默认提交约定的有效提交类型。你可能`fix: fix a bug`想要

Thank you. That's true
